Scripts/BuildDatabank/makeRanking.py

```diff
@@ -5,8 +5,8 @@
 
 import sys
 
-sys.path.insert(1, '../BuildDatabank/')
-from databankLibrary import *
+sys.path.append('..')
+from DatabankLib.databankLibrary import *
```

Comment:
to have an internal consistency in importing databank related modules and methods, could you please explicitly define the imported methods here, instead of using *?

Reply:
done during style alignment.